contracts/shared/TestUniswapRouter.sol

```diff
@@ -0,0 +1,222 @@
+pragma solidity =0.6.6;
```

Comment:
why 0.6? we are using 0.8, beside uniswap has a lower version we may try to upgrade it